tests/AppBundle/Controller/PagesControllerTest.php

```diff
@@ -23,7 +23,7 @@ public function testIndex()
         $crawler = $client->request('GET', '/pages');
         $this->assertEquals(200, $client->getResponse()->getStatusCode());
 
-        if ($this->container->getParameter('app.is_labs')) {
+        if ($this->container->getParameter('app.is_labs') && !$this->container->getParameter('app.single_wiki')) {
```

Comment:
We should really encapsulate isLabs somewhere else, because it should never be single wiki if it's labs. But this is good for now.